index.html

```diff
@@ -64,8 +64,11 @@
       <br>
       <a href="#" class="[ button ][ ffab before remove fa-arrow-left ]">Transition Remove</a>
       <a href="#" class="[ button ][ ffab remove fa-arrow-right ]">Transition Remove</a>
+      <hr>
+      <strong>Hover/Active State</strong>
       <br>
-      <a href="#" class="[ button ][ ffab static fa-arrow-right ]">No Transition (Static)</a>
+      <a href="#" class="[ button ][ ffab fa-arrow-right ]">With Hover/Active</a>
+      <a href="#" class="[ button ][ ffab fa-arrow-right no-hover-active ]">Without Hover/Active</a>
```

Comment:
The short answer, yes I would definitely want to be able to add one or the other, though not necessarily on the individual level.  The long answer, see below...